From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 23 Feb 2018 14:00:56 +0000
Subject: sfc: falcon: remove duplicated bit-wise or of LOOPBACK_SGMII
Patch-mainline: v4.17-rc1
Git-commit: 9baeb5eb1f83b8c64a80ee9926204909e275b5cc
References: bsc#1105555 FATE#326117

Bit pattern LOOPBACK_SGMII is being bit-wise or'd twice; remove the
redundant 2nd LOOPBACK_SGMII

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/sfc/falcon/enum.h |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/sfc/falcon/enum.h
+++ b/drivers/net/ethernet/sfc/falcon/enum.h
@@ -81,7 +81,6 @@ enum ef4_loopback_mode {
 			    (1 << LOOPBACK_XAUI) |		\
 			    (1 << LOOPBACK_GMII) |		\
 			    (1 << LOOPBACK_SGMII) |		\
-			    (1 << LOOPBACK_SGMII) |		\
 			    (1 << LOOPBACK_XGBR) |		\
 			    (1 << LOOPBACK_XFI) |		\
 			    (1 << LOOPBACK_XAUI_FAR) |		\
